scripts/fuzz.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -e
+
+function build_fuzz() {
+  PACKAGES=$1
+  
+  for pkg in $PACKAGES; do
+    pushd fuzz/$pkg
+  
+    for file in *.go; do
+      if [[ "$file" == "fuzz_utils.go" ]]; then
+        continue
+      fi
+  
+      NAME=$(echo $file | sed 's/\.go$//1')
+      echo "Building zip file for $pkg/$NAME"
+      go-fuzz-build -func "Fuzz_$NAME" -o "$pkg-$NAME-fuzz.zip" "github.com/lightningnetwork/lnd/fuzz/$pkg"
+    done
+  
+    popd
+  done
+}
+
+# timeout is a cross platform alternative to the GNU timeout command that
+# unfortunately isn't available on macOS by default.
+timeout() {
+    time=$1
+    $2 &
+    pid=$!
+    sleep $time
+    kill -s SIGINT $pid    
+}
+
+function run_fuzz() {
+  PACKAGES=$1
+  RUN_TIME=$2
+  TIMEOUT=$3
+  PROCS=$4
+  BASE_WORKDIR=$5
+  
+  for pkg in $PACKAGES; do
+    pushd fuzz/$pkg
```

Comment:
Not sure why it happens but this now doesn't work with `lnwire` after running `brontide`:

```
Running fuzzer brontide-static_resp_encrypt-fuzz.zip with 4 processors for 30 seconds
go-fuzz -bin=brontide-static_resp_encrypt-fuzz.zip -workdir=/Users/nsa/Desktop/lnd_fuzz_seeds/brontide/static_resp_encrypt -procs=4 -timeout=20
2020/10/08 17:46:01 workers: 4, corpus: 4 (3s ago), crashers: 0, restarts: 1/0, execs: 0 (0/sec), cover: 0, uptime: 3s
2020/10/08 17:46:04 workers: 4, corpus: 4 (6s ago), crashers: 0, restarts: 1/0, execs: 0 (0/sec), cover: 0, uptime: 6s
2020/10/08 17:46:07 workers: 4, corpus: 4 (9s ago), crashers: 0, restarts: 1/1, execs: 4 (0/sec), cover: 551, uptime: 9s
2020/10/08 17:46:10 workers: 4, corpus: 4 (12s ago), crashers: 0, restarts: 1/1, execs: 4 (0/sec), cover: 551, uptime: 12s
2020/10/08 17:46:13 workers: 4, corpus: 4 (15s ago), crashers: 0, restarts: 1/501, execs: 4015 (268/sec), cover: 551, uptime: 15s
2020/10/08 17:46:16 workers: 4, corpus: 4 (18s ago), crashers: 0, restarts: 1/1213, execs: 9706 (539/sec), cover: 551, uptime: 18s
2020/10/08 17:46:19 workers: 4, corpus: 4 (21s ago), crashers: 0, restarts: 1/1879, execs: 15037 (716/sec), cover: 551, uptime: 21s
2020/10/08 17:46:22 workers: 4, corpus: 4 (24s ago), crashers: 0, restarts: 1/2484, execs: 19874 (828/sec), cover: 551, uptime: 24s
2020/10/08 17:46:25 workers: 4, corpus: 4 (27s ago), crashers: 0, restarts: 1/3251, execs: 26014 (963/sec), cover: 551, uptime: 27s
2020/10/08 17:46:28 shutting down...
scripts/fuzz.sh: line 43: pushd: fuzz/lnwire: No such file or directory
make: *** [fuzz-run] Error 1
```

Reply:
Oops, forgot a `popd` command. Tested it again locally, works now.